worknenjoy/gitpay
# Stats page UI Improvement

## Description

> Previously ugly looking stats page is redefined to look more elegant 

## Changes

- changes apply to the Stats page

## Issue

> Closes #677 

## Impacted Area

> Stats page

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- simply head toward the {url}/stats link to see the stats page

## Before
> Screenshot from the state before

## After
> 
![image](https://user-images.githubusercontent.com/43280610/88490739-62b3b580-cfbb-11ea-854c-3f12eff3567b.png)
